engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java

```diff
@@ -920,8 +927,8 @@ private void deployNodes(List<OperatorDeployInfo> nodeList) throws IOException
   }
 
   private HashMap<String, ComponentContextPair<Stream, StreamContext>> deployOutputStreams(
-    List<OperatorDeployInfo> nodeList, HashMap<String, ArrayList<String>> groupedInputStreams)
-    throws Exception
+      List<OperatorDeployInfo> nodeList, HashMap<String, ArrayList<String>> groupedInputStreams)
```

Comment:
@tweise @chandnisingh What would you want this to look like? Is it with regards to ```throws Exception``` being on the new line? This seems to be an IDE Settings issue, code style does not adjust it one way or the other. The relevant setting seems to be ```throws keyword``` to ```wrap if long``` but it makes no difference. I think from a style perspective both are valid.